src/main/java/com/wynntils/modules/map/events/ClientEvents.java

```diff
@@ -123,6 +123,11 @@ public void guiOpen(GuiOverlapEvent.ChestOverlap.InitGui e) {
         if (LootRunManager.isRecording())
             LootRunManager.addChest(lastLocation); // add chest to the current lootrun recording
 
+        if(LootRunManager.isLootrunLoaded()){
```

Comment:
```suggestion
        if (LootRunManager.isLootrunLoaded()) {
```
Please be consistent with your formatting.